### PR TITLE
Changes to several sections

### PR DESCRIPTION
**Pull request by camelkk**
I changed some previous sections I edited and went through the rest of the whitepaper,
not sure if the formatting is correct but I didn't add too much so it should be okay.
thanks

### CODE REVIEW

File: whitepapers/Viacoin_whitepaper.tex [hunk 1]
```diff
@@ -135,17 +135,17 @@ \section{The Lightning Network}\label{sec: The Lightning Network}
 become a focal point for routing the payment of others leading into a fully
 connected payment channel. The payments are enforced using a script which enforces the atomicity via decrementing time-locks.
 \newline \newline \noindent
-Another benefit is the possibility of atomic cross-chain transactions, enabling users to trade viacoin, bitcoin, litecoin and other Segwit coins
+Another benefit is the possibility of atomic cross-chain transactions, enabling users to trade Viacoin, Bitcoin, Litecoin and other Segwit coins
```
Comment (romanornr) [inline whitepapers/Viacoin_whitepaper.tex]: Bitcoin refers to the Bitcoin protocol
bitcoin refers to the currency itself.

I think they should stay lowercase

**Review by romanornr — CHANGES_REQUESTED** [review]
Thanks for the help. I agree with the changes except for the caps. (see review) 

I think they should stay as lowercase.

File: whitepapers/Viacoin_whitepaper.tex (hunk 1 above)
Comment (camelkk) [reply whitepapers/Viacoin_whitepaper.tex]: ah fair enough, sounds good

**Comment by camelkk** [comment]
Updated the capitalization of litecoin, viacoin and bitcoin, thanks for the heads up.
Not 100% if I did it correctly, new to github.

**Comment by romanornr** [comment]
ACK ! 
Merged & compiling went fine. 

Thanks for the help !

**Pull request merged**